### ISSUE TITLE
Allow for a soccer game to be played programmatically

### ISSUE THREAD

**Issue by JNeiger** [RoboJackets/robocup-software#841]
It would be nice to be able to specify a play (Or set of plays) to be played for both yellow and blue teams. In the end, maybe return some sort of struct or object with the score, and some other variables.

There are already [auto-ref systems](https://github.com/robotics-erlangen/autoref) setup so I think just setting up soccer to programmatically start and end games will be good enough.

This would be the start of automatically tuning play constants and doing machine learning in the future.

**Comment by jpfeltracco** [comment]
Yeah, it would be great if we could push towards being able to simulate full matches to allow us to objectively say one play is "better" than another, especially if we want new members to have a play competition next semester.

We talked a bit in the meeting today about using a more maintained version of grSim that would integrate with the autoref and let this happen. @jgkamat may know which team has a more maintained version that we could take a look at.
